README.md

```diff
@@ -129,4 +131,4 @@ Contain simple API wrappers.
 Wrapper over `zap.Logger` which is used across NeoFS codebase.
 
 ### util
-Utilities for working with signature-related code.
\ No newline at end of file
+Utilities for working with signature-related code.
```

Comment:
unrelated to commit?

Reply:
A typical `\ No newline at end of file` thing. Usually I leave them as is.